cli/codegen/static/static.go

```diff
@@ -0,0 +1,17 @@
+// +build dev
+
+package static
+
+//go:generate go run -tags=dev ../generate_vfs.go -o templatexxx.go
+
+import (
+	"net/http"
+	"path/filepath"
+)
+
+var CartridgeTemplateFS http.FileSystem = http.Dir(filepath.Join("..", "..", "create", "templates", "cartridge"))
+var AdminLuaTemplateFS http.FileSystem = http.Dir(filepath.Join("..", "..", "admin", "templates"))
```

Comment:
Think about

`CartridgeTemplateFS` -> `CreateCartridgeTemplateFS`
`AdminLuaTemplateFS` -> `AdminLuaCodeFS` 
`ConnectorLuaTemplateFS` -> `ConnectorLuaCodeFS`

and so on.

I like using sub-package name as a first part (`Create`, `Admin`, ...), but `CreateCartridgeTemplateFS` is application template when `ConnectLuaTemplateFS` doesn't contain templates as well, it contains Lua code that should be evaled on instance.

Moreover, I don't like using `templates` in paths like `filepath.Join(..., "admin", "templates")`. Templates is used for application templates, but other such dirs contains Lua code to be evaled (some of this code are templates, but it should be reflected in file name, not directory name).

About templated code - this templates aren't computed in runtime (see `getAdminFuncEvalTypedBody` for example, or using of `formatTopologyReplicasetFunc`), all these values can be computed  on `generate`.